docassemble/GithubFeedbackForm/github_issue.py

```diff
@@ -85,6 +87,17 @@ def feedback_link(user_info_object=None,
     session_id=_session_id,
     local=False,reset=1)
 
+def add_panel_participant(email):
+    """Adds this email to a list of potential participants for a qualitative research panel.
+    Only adds the email, as we shouldn't link their feedback to their identity at all.
+    """
+    red = DARedis()
+    red.sadd(redis_panel_emails_key, email)
```

Comment:
Quick suggestion: can we add the date as well? It may be helpful to filter panels by how recently they volunteered.